### ISSUE TITLE
Getting "File name too long" when move file with russian name

### ISSUE THREAD

**Issue by mico** [instructure/canvas-lms#676]
At files course page like http://mydomain/courses/1/files when i try to delete file with russian name like "Качество образования-1.pptx" i'm getting error:
```
Errno::ENAMETOOLONG (File name too long - /tmp/attachment_fu/20150821-13041-1cdr
vyq8534860431440182728_622__%2525D0%25259A%2525D0%2525B0%2525D1%252587%2525D0%25
25B5%2525D1%252581%2525D1%252582%2525D0%2525B2%2525D0%2525BE%252B%2525D0%2525BE%
2525D0%2525B1%2525D1%252580%2525D0%2525B0%2525D0%2525B7%2525D0%2525BE%2525D0%2525B2%2525D0%2525B0%2525.pptx.lock):
  /home/mico/canvas/gems/activesupport-suspend_callbacks/lib/active_support/callbacks/suspension.rb:111:in `instance_eval'
  /usr/lib/ruby/1.9.1/tempfile.rb:346:in `rmdir'
  /usr/lib/ruby/1.9.1/tempfile.rb:338:in `ensure in locking'
  /usr/lib/ruby/1.9.1/tempfile.rb:338:in `locking'
  /usr/lib/ruby/1.9.1/tempfile.rb:144:in `block in initialize'
  /home/mico/canvas/config/initializers/ruby_version_compat.rb:24:in `create'
  /usr/lib/ruby/1.9.1/tempfile.rb:134:in `initialize'
  /home/mico/canvas/gems/attachment_fu/lib/attachment_fu.rb:216:in `new'
  /home/mico/canvas/gems/attachment_fu/lib/attachment_fu.rb:216:in `copy_to_temp_file'
```
in log and error message at page.

Also i cannot move file to another folder.

But upload and download working.

**Comment by rivernate** [comment]
Thanks for reporting this, I was able to duplicate it, and created an internal ticket for it.

**Comment by rivernate** [comment]
This has been fixed here 82c3bf7f3d7823f00cb38bee754cfcfd3276f89e
